### ISSUE TITLE
⌨ "Résumé" improperly announced to English ATs

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#7013]
## ⌨ Accessibility Issue

We currently use the term "résumé" on the English version of the application. This uses a borrowed word and contains accents, the assistive technology does not recognize the accents and reads out "resume" which is very confusing.

### WCAG 2.1 Criteria

This violates [SC 3.1.6 Pronunciation](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-pronunciation.html). 

## 🦋 Expected Behaviour

An acceptable technique is to use a word that does not have an ambiguous spelling or pronunciation. 

## 🕵️ Details

Add any additional details that could assist with troubleshooting/fixing the issue.

- **Operating System**: MacOS 13.4 & Windows 11.
- **Browser**: Firefox 15.0b7, Google Chrome 114.0.5735.133
- **Assistive Technology**: VoiceOver, NVDA, JAWS

## 📋 Steps to Reproduce

1. Navigate to `/applicant/dashboard`
2. Have the word résumé be read out by a screen reader
3. Confirm it is mispronounced

## 🙋‍♀️ Proposed Solution

Use an alternate term such as CV or provide a proper pronunciation of the word.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] résumé is replaced with an unambiguous term or;
- [ ] résumé has a provided pronunciation for ATs

**Comment by mnigh** [comment]
> * résumé has a provided pronunciation for ATs

I am not in favour of this idea as it:

1. sets a bad precedent for sidestepping the use of plain language
2. creates another point of failure if phonetic spellings are treated differently by each assistive technology

**Comment by RM-1978** [comment]
@mnigh and @substrae,  Dasha and I explored other options (e.g., CV, Curriculum Vitae, Work History)  and the simplest PL solution would simply be to call this section by the nature of its contents: "Education, Qualifications and Experience" 

That said, this does raise the larger question for @substrae, @gobyrne and @lauren about whether a candidate profile container should be frames as both the candidate account details and preferences, as well as their skill and experience history.

**Comment by RM-1978** [comment]
@substrae 

Sounds good. Let's chat with her on Thursday :)

**Comment by mnigh** [comment]
This issue will be resolved if #7277 is completed.

**Comment by esizer** [comment]
> This issue will be resolved if #7277 is completed.

Should we attach this to your PR or just close it in favour of the ticket you are working on?

**Comment by mnigh** [comment]
> > This issue will be resolved if #7277 is completed.
> 
> Should we attach this to your PR or just close it in favour of the ticket you are working on?

Once #7277 is complete, we can revisit this issue and decide if it still is relevant. Technically, there will still be a few instances of résumé that are rendered on the IAP home page, but those will not be changed as far as I know.

**Comment by mnigh** [comment]
Resolved in https://github.com/GCTC-NTGC/gc-digital-talent/issues/7277.
